tests/widgets_paragraph.rs

```diff
@@ -364,3 +364,18 @@ fn widgets_paragraph_can_align_spans() {
         ]),
     );
 }
+
+#[test]
+fn widgets_paragraph_counts_rendered_lines() {
```

Comment:
can you please add some tests for the lines that codecov annotates as not covered (width < 1, no wrap and line_width function)